pyproject.toml

```diff
@@ -20,11 +20,18 @@ profile = "black"
 line_length = 100
 [build-system]
 requires = [
+    "pyxdg>=0.26",
     "requests>=2.24",
+    "setuptools>=46.4",
+    "sphinx",
     "toml>=0.10.2",
     "urllib3>=1.24.2",
-    "pyxdg>=0.26",
-    "setuptools>=46.4",
     "wheel",
 ]
```

Comment:
Minor-nit: It appears (maybe I'm wrong) the `requires` block needs to match `requirements.txt`?  These are the kinds of things which will drive developers crazy if they ever get out of sync for some reason.   Short of some kind of script/automation, consider adding comments to both files indicating developers changing one, should always also change the other.  It's kind of a "cheat" but may save someone hours of debugging.